### PR TITLE
PUBDEV-8254: H2O shouldn't run as a root user in example Dockerfile

### PR DESCRIPTION
**Pull request by michalkurka**

### CODE REVIEW

**Review by g-eoj — COMMENTED** [review]
Need to understand how the working directory is intended to be used. Is this image just for h2o internal testing or are we intending it to be more universal?

File: Dockerfile [hunk 1]
```diff
@@ -22,33 +22,41 @@ RUN \
...
 
 # Define a mountable data directory
 #VOLUME \
 #  ["/data"]
 
 # Define the working directory
-#WORKDIR \
-#  /data
+WORKDIR \
+  /home/jenkins
```
[the remark this answers is not in the document]
Comment (michalkurka) [reply Dockerfile:59]: good catch, it is a typo - it should be h2o, it probably doesn't do anything in the image

**Comment by michalkurka** [comment]
@g-eoj it is not meant for testing, it doesn't have any other than a legacy purpose

its value it that it starts H2O in a docker image without needing to think about memory setting

**Review by g-eoj — APPROVED** [review]

**Comment by michalkurka** [comment]
thanks for catching the issue and reviewing @g-eoj

**Pull request merged**